openam-ui/openam-ui-ria/src/main/resources/locales/ja/translation.json

```diff
@@ -0,0 +1,1291 @@
+                "addModuleError": "認証インスタンスがすでに存在しています。",
+                "selectModuleType" : "モジュールタイプの選択   ",
+                "addModule" : "モジュールの追加",
+                "moduleName" : "モジュール名",
+                "moduleType" : "タイプ",
+                "chain" : "認証連鎖",
+                "inUse": {
+                    "title": "モジュールの編集",
+                    "message": "<code>__moduleName__</code> モジュールは、認証連鎖 <code>__usedChains__</code> で使用されています。このモジュールを編集すると、その動作が変更することがあります。<br><br>継続してもよろしいですか?"
+                },
+                "moduleNameValidationError" : "モジュールの名前に空白は許可されていません。",
+                "confirmDeleteSelected": "選択したモジュールを削除してもよろしいですか?",
+                "confirmDeleteSelected_plural": "選択したモジュールを削除してもよろしいですか?"
+            },
+            "chains": {
+                "title" : "認証連鎖",
+                "newChain" : "認証連鎖の追加",
+                "chainName" : "認証連鎖名",
+                "description" : "対象を認証する手順を定義するために、認証連鎖を使用してください。認証連鎖は連続する1つ以上の認証モジュールで構成され、資格情報は認証連鎖の次のモジュールインスタンスに受け継がれます。",
+                "defaultAdminAuthConfiguration" : "デフォルトの管理者の認証連鎖",
+                "defaultOrgAuthConfiguration" : "デフォルトの組織の認証連鎖",
+                "duplicateChain" : "同じ名前の認証連鎖がすでに存在します。",
+                "confirmDeleteSelected": "選択した認証連鎖を削除してもよろしいですか?",
+                "confirmDeleteSelected_plural": "選択した認証連鎖を削除してもよろしいですか?"
+            },
+            "editChains": {
+                "options" : "オプション",
+                "noOptions" : "オプション無し",
+                "selectCriteria" : "基準の選択",
+                "addValue" : "値の追加",
+                "selectModule" : "モジュールの選択",
+                "chains" : "認証連鎖",
+                "editChain" : "認証連鎖の編集",
+                "newModule" : "新規モジュール",
+                "addModule": "モジュールの追加",
+                "editModule" : "モジュールの編集",
+                "gotoNextModule" : "次のモジュールへと通過",
+                "gotoSuccessUrl" : "ログイン成功 URL に転送",
+                "gotoFailUrl" : "ログイン失敗 URL に転送",
+                "redirectUrls" : "リダイレクト URL",
+                "redirectUrlsHelp" : "対象がリダイレクトされる URL を指定します。認証失敗時は、認証成功時と異なる URL を指定することができます。",
+                "successUrl" : "ログイン成功 URL",
+                "failUrl" : "ログイン失敗 URL",
+                "addUrl" : "URL の追加",
+                "dropHere" : "ここにドロップ",
+                "moduleHelp" : "すべてのモジュールインスタンスにユーザーの資格情報を渡すプロセスを構築するように、認証モジュールを追加してください。",
+                "processingClass": {
+                    "title": "ポスト認証プロセスクラス",
+                    "className": "クラス名",
+                    "addClassNameDialog": {
+                        "title": "新しいクラス名の追加",
+                        "message": "ポスト認証プロセスの新規クラス <mark>__newClassName__</mark> は、保存されていません。保存しますか?"
+                    },
+                    "duplicateClass" : "同じ名前のクラスがすでに存在します",
+                    "help" : "認証プロセス終了時に実行する Java クラスの名前を指定します。"
+                },
+                "criteria" : [
+                    {
+                        "title": "必須",
```

Comment:
Certainly I was often confused with these selections.
But It is easy to understand with the arrow in GUI, and there are hint balloons, so we will not worry about mistaking the operation.

If I express their behavior, my proposal is as follows:

- authentication.config.required.label  -> 必須(失敗時継続)
- authentication.config.requisite.label -> 必須(失敗時中断)

Do you have any idea?

Reply:
I think that the proposal is easier to understand than the current translation. If you commit this content, I will approve it.

However, personally, these words are technical terms, so I think that it is better to leave these words in English.

Reply:
I agree that you pointed out that it is a technical term.

However I think it is not easy to choose the right one because:
 - People of various roles (including end users, SI team) are use this UI.
 - When multiple languages mixed in the dropdown list, Users may think that it is strange or uncomfortable.

So, I will ask my colleagues for opinions.
Please give me little more time.

Reply:
> When multiple languages mixed in the dropdown list, Users may think that it is strange or uncomfortable.

I'm sorry I made you misunderstand.
I think that it is better to leave `Optional` and `Sufficient` in English as well.
In that case, multiple languages will not be mixed in the dropdown list.

> So, I will ask my colleagues for opinions.

I appreciate it.

Reply:
I'm so sorry for my late reply.

I agree with the policy to leave criteria items in English.
My colleague's opinion was that it is difficult to choose the right item by looking at the item name in any Japanese translation.

Reply:
Thank you for your consideration.

> it is difficult to choose the right item by looking at the item name in any Japanese translation.

That's true.